docs/node_scenarios.md

```diff
@@ -13,11 +13,13 @@ Following node chaos scenarios are supported:
 
 **NOTE**: If the node doesn't recover from the node_crash_scenario injection, reboot the node to get it back to Ready state.
 
-**NOTE**: node_start_scenario, node_stop_scenario, node_stop_start_scenario, node_termination_scenario, node_reboot_scenario and stop_start_kubelet_scenario are supported only on AWS as of now.
+**NOTE**: node_start_scenario, node_stop_scenario, node_stop_start_scenario, node_termination_scenario, node_reboot_scenario and stop_start_kubelet_scenario are supported only on AWS and Azure as of now.
```

Comment:
Before the list of NOTEs, document that the valid values for cloud_type are now aws, gcp and az